src/Controls/src/Core/Platform/AlertManager/AlertManager.iOS.cs

```diff
@@ -206,6 +215,20 @@ static void PresentPopUp(Window virtualView, UIWindow platformView, UIAlertContr
 					_ = platformView.RootViewController.PresentViewControllerAsync(alert, true);
 				});
 			}
+
+			bool PageIsInThisWindow(IView view)
+			{
+				if (view is not Page page)
+					return false;
+
+				var window = page.Window;
+				var platformWindow = window?.MauiContext.GetPlatformWindow();
```

Comment:
if window is null then MauiContext will be null therefor you will get a NRE  in that case when calling GetPlatformWindow, correct? I think we need to add an extra ?
```suggestion
				var platformWindow = window?.MauiContext?.GetPlatformWindow();
```

Reply:
I see in the code we have the MauiContext never null because it will throw if that is the case. And GetPlatformWindow is also never null as it also throws.